tests/integration/create.bats

```diff
@@ -39,3 +39,19 @@ function teardown() {
 
   testcontainer test_busybox running
 }
+
+@test "runc create --pid-file" {
+  runc create --pid-file pid.txt --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  testcontainer test_busybox created
+
+  # check pid.txt was generated
+  [ -e pid.txt ]
```

Comment:
I thought for a moment.  the `pid` is sent by C program to Go program and write to file by Go. It is necessary to check it matches. I update the patch with